README.md

```diff
@@ -19,7 +19,11 @@ seems to be around 3x slower than implementations using Torch.
 
 `python neural_style.py --content <content file> --styles <style file> --output <output file>`
 
-(run `python neural_style.py --help` to see a list of all options)
+Run `python neural_style.py --help` to see a list of all options.
+
+Use `--checkpoint-output` and `--checkpoint-iterations` to save checkpoint images.
+
+Use `--iterations` to change the number of iterations (default 1000).  1000 iterations take around 10 hours to run on a Mid 2013 Macbook Air for a 528×960 pixel content file.  
```

Comment:
Can we mention something about a GPU here? This program should really be run on a machine with a powerful GPU 😛 

Benchmarking on a Maxwell Titan X or a Pascal Titan X seems like it would be good.